src/Features/VisualBasic/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.vb

```diff
@@ -280,12 +280,89 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.InlineTemporary
             Return localDeclaration.Parent
         End Function
 
+        Private Function GetNonWhitespaceTrivia(trivia As IEnumerable(Of SyntaxTrivia)) As IEnumerable(Of SyntaxTrivia)
+            Dim newLeadingTrivia = trivia _
+                .Reverse _
+                .SkipWhile(Function(t) t.IsKind(SyntaxKind.WhitespaceTrivia, SyntaxKind.EndOfLineTrivia)) _
+                .Reverse
+
+            ' Ensure that we leave a line break if trivia with a comment.
+            If newLeadingTrivia.Any() AndAlso newLeadingTrivia.Last().IsKind(SyntaxKind.CommentTrivia) Then
```

Comment:
super expensive calls.  both Any and Last here end up re-realizing the lazy stream.